config/auth.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Authentication Defaults
+    |--------------------------------------------------------------------------
+    |
+    | This option controls the default authentication "guard" and password
+    | reset options for your application. You may change these defaults
+    | as required, but they're a perfect start for most applications.
+    |
+    */
+
+    'defaults' => [
```

Comment:
The Lumen module has a development status. And I am not really inclined to wait for the next major release of Codeception after putting in this much work for what I am sure is an improvement of the module.